### ISSUE TITLE
[Android] "No result returned" errors

### ISSUE THREAD

**Issue by donatoaguirre24** [plaid/react-native-plaid-link-sdk#419]
## Empty errors in production

Along with normal errors like `INTERNAL_SERVER_ERROR` or `insititution_not_found`, we're seeing many errors with "No result returned" in production in the link bank account flow.

## Environment

- Plaid Link React Native: 7.1.0
- ReactNative Version: 0.65.1
- Occurs on Android: yes
- Occurs on iOS: no             

## Expected Result

A meaningful error or documentation explaining why the error is empty and what could have gone wrong.

## Logs

This is the error we're seeing

```JSON
{
  "metadata": {},
  "error": {
    "errorMessage": "No result returned.",
    "errorCode": "No result returned.",
    "displayMessage": "No result returned.",
    "errorDisplayMessage": "No result returned."
  }
}
```

**Comment by amytang0** [comment]
This occurs when the Android Activity result returned is null. This can occur when the Android OS cleans up the Link Activity in the background. Since React Native only runs one background thread, this can occur more frequently on RN. How often are you running into this error?

**Comment by amytang0** [comment]
Hi sorry for the late response; we have patched this in `7.2.1` and will be fixing it for Headless EU configs in next month's release
